src/test/scala/com/microsoft/hyperspace/index/RefreshIndexDeleteTests.scala

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (2020) The Hyperspace Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.microsoft.hyperspace.index
+
+import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.fs.Path
+import org.apache.spark.sql.catalyst.plans.SQLHelper
+
+import com.microsoft.hyperspace.{Hyperspace, HyperspaceException, SampleData}
+import com.microsoft.hyperspace.util.FileUtils
+
+class RefreshIndexDeleteTests extends HyperspaceSuite with SQLHelper {
```

Comment:
General comment: Can you please add class and method comments?

Reply:
Sure, I updated the PR description and added comments for `RefreshDeleteAction` class.